### PR TITLE
feat(contact): add contact tab

### PR DESCRIPTION
**Pull request by ncoden**
Open a contact tab when an item in "contacts" list is clicked.

Require:
* [x] https://github.com/CaliOpen/caliopen.api.user/issues/5

Preview:
preview

### CODE REVIEW

File: src/js/action/action-types.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ export const RECEIVER_MESSAGES = 'RECEIVER_MESSAGES';
 export const REQUEST_THREADS = 'REQUEST_THREADS';
 export const RECEIVER_THREADS = 'RECEIVER_THREADS';
 
+export const REQUEST_CONTACT = 'REQUEST_CONTACT';
+export const RECEIVER_CONTACT = 'RECEIVER_CONTACT';
```
Comment (iamdey) [inline src/js/action/action-types.js]: I think I made a mistake when I wrote the other actions, I don't remember what I meant by "receiver". 
We'll change in a refactor

File: src/js/action/contacts.js [hunk 2]
```diff
@@ -6,6 +6,23 @@ export class ContactsActions {
...
 
+  requestContact(contactId) {
+    return {
+      type: action.REQUEST_CONTACT,
+      contactId
+    };
+  }
+
+  receiveContact(contactId, json) {
+    console.log(json);
```
Comment (iamdey) [inline src/js/action/contacts.js]: log

**Comment by iamdey** [comment]
:+1:

File: src/js/action/action-types.js (hunk 1 above)
Comment (ncoden) [inline src/js/action/action-types.js]: Request: prepare the HTTP request
Receiver: processes the HTTP request results
Don't know if it's what you search.

File: src/js/action/contacts.js (hunk 2 above)
Comment (ncoden) [inline src/js/action/contacts.js]: oups

**Comment by ncoden** [comment]
Fixed. Please merge if it's ok.

File: src/js/action/action-types.js (hunk 1 above)
Comment (iamdey) [inline src/js/action/action-types.js]: nope. I mean `receiver` itself is nonsense

**Pull request merged**